eng/pipelines/templates/jobs/sdk-build.yml

```diff
@@ -49,10 +49,18 @@ jobs:
     enableSbom: ${{ parameters.enableSbom }}
     variables:
     - ${{ insert }}: ${{ parameters.variables }}
+    templateContext:
+      sdl:
+        binskim:
+          analyzeTargetGlob: +:f|artifacts\bin\**\*.dll;+:f|artifacts\bin\**\*.exe;-:f|artifacts\bin\**\msdia140.dll;
 
     steps:
     ############## PREP ###############
     - ${{ if eq(parameters.publishRetryConfig, true) }}:
+      # Pre-create artifacts/bin to avoid errors in Guardian globbing
+      - powershell: New-Item -ItemType Directory -Path $(Build.SourcesDirectory)/artifacts/bin -Force
+        displayName: Create artifacts/bin directory
```

Comment:
This isn't the preferred way to do this. It won't work on non-Windows builds, and isn't necessary for non-Windows builds. When testing this, I did it on Marc's internal branch and added a preStep to the Windows-only job definition: https://dnceng.visualstudio.com/internal/_git/dotnet-sdk?path=/.vsts-ci.yml&version=GBmarcpopMSFT-enable1essdl&line=112&lineEnd=116&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents